docs/source/cite.rst

```diff
@@ -0,0 +1,22 @@
+Citations
+=========
+
+Dask is developed by many people from many institutions.  Some of these
+developers are academics who depend on academic citations to justify their
+efforts.  Unfortunately, no single citation can do all of these developers (and
+the developers to come) sufficient justice.  Instead, we choose to use a single
+blanket citation for all developers past and present.
+
+To cite Dask in publications, please use the following::
+
+   Dask Development Team (2016). Dask: Library for dynamic task scheduling
+   URL http://dask.pydata.org
+
+A BibTeX entry for LaTeX users follows::
+
+   @Manual{,
+     title = {Dask: Library for dynamic task scheduling},
+     author = {{Dask Development Team}},
+     year = {2016},
+     url = {http://dask.pydata.org},
+   }
```

Comment:
I'm going to use this entry for now unless otherwise corrected.  Thanks @mrocklin!